<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creo_cartpage</class>
 <widget class="QMainWindow" name="creo_cartpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: rgb(255, 87, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QFrame" name="title_frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>49</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px; 
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="back_btn">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 15px; 
border: 1px solid rgb(255, 99, 101);
font: 8pt &quot;Segoe UI Black&quot;;
color: red;</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 87 14pt &quot;Segoe UI Black&quot;</string>
         </property>
         <property name="text">
          <string>Shopping Cart</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>55</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="remove_btn">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 15px; 
background-color: rgb(255, 99, 101);
font: 8pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="search_frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>49</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255); border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLineEdit" name="search_bar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #ccc;           
    border-radius: 18px;           
    padding: 10px;                   
    font-size: 16px;           
    color: #333;      
    background-color: #fff;
	font: 10pt &quot;Segoe UI&quot;;
}

QLineEdit:focus {
    border: 1px solid #0078d7; 
    background-color: #f0f8ff;       
}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>What to search?</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="search_btn">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 15px; background-color: rgb(170, 255, 255);font: 8pt &quot;Segoe UI Black&quot;; color: skyblue;</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="product_list_frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255); border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="product_table">
         <property name="minimumSize">
          <size>
           <width>471</width>
           <height>361</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget {
    border: none;                
    gridline-color: #e0e0e0;          
    background-color: #ffffff;}         

QTableWidget::item {
    padding: 10px;                   
    border: none;                    
}

QTableWidget::item:selected {
    background-color: #0078d7;       
    color: white;                     
}

QHeaderView {
    background-color: #f7f7f7;       
    border: none;                     
    font-size: 14px;              
    padding: 5px;                 
}

QHeaderView::section {
    padding: 10px;                     /* Padding inside header sections */
    border: none;                      /* Remove section borders */
    color: #333;                       /* Header text color */
}

QTableWidget::horizontalHeader {
    border-bottom: 1px solid #e0e0e0; /* Bottom border for header */
}

QTableWidget::verticalHeader {
    border-right: 1px solid #e0e0e0;  /* Right border for vertical header */
}
</string>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed</set>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>137</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Product</string>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignTop</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="checkout_frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>49</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255); border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="price_text">
         <property name="styleSheet">
          <string notr="true">font: 87 16pt &quot;Segoe UI Black&quot;;</string>
         </property>
         <property name="text">
          <string>(0) PX.XX</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="checkout_btn">
         <property name="minimumSize">
          <size>
           <width>211</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 15px; font: 8pt &quot;Segoe UI Black&quot;; 
background-color: rgb(255, 11, 117);
color: white;</string>
         </property>
         <property name="text">
          <string>Checkout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
